Cargo.toml

```diff
@@ -39,6 +39,7 @@ serde = { version = "1", optional = true }
 serde_json = { version = "1", optional = true }
 encoding = { version = "0.2", optional = true }
 native-tls = { version = "0.2", optional = true }
+cookie_store = "0.12.0"
```

Comment:
This needs to be optional on the cookie feature or we pull in the deps for all features.